libr/core/cbin.c

```diff
@@ -1653,6 +1658,7 @@ static int bin_imports(RCore *r, int mode, int va, const char *name) {
 		r_cons_println ("fs imports");
 	} else if (IS_MODE_NORMAL (mode)) {
 		r_cons_println ("[Imports]");
+		r_cons_println (" Num Vaddr       Bind      Type Name");
```

Comment:
why the initial space before `Num`?

Reply:
In my testing I hadn't seen a number larger than three digits here, so was trying to line up with the 2 and three digit numbers nicely. Looking at the source again, a 4-digit number is possible, so will change this.

Reply:
I asked because in other cases (e.g. exports, symbols, etc.) you aligned `Num` to the left, without spaces.

Reply:
Yeah, in all the headers I tried to align with the leftmost characters I could see, where this was possible. With this field it would be possible to have a 4-digit number, in which case it would make sense to remove the space.